src/components/asset-list/RecyclerAssetList2/index.tsx

```diff
@@ -17,9 +17,11 @@ function RecyclerAssetList() {
 
   const position = useMemoOne(() => new RNAnimated.Value(0), []);
 
+  const value = useMemo(() => ({ additionalData }), [additionalData]);
+
   return (
     <RecyclerAssetListScrollPositionContext.Provider value={position}>
-      <RecyclerAssetListContext.Provider value={additionalData}>
+      <RecyclerAssetListContext.Provider value={value}>
```

Comment:
these files are looking fine to me but I think we need review from someone that actually worked on this before, maybe once we push this to develop